src/components/Navigation/Navigation.tsx

```diff
@@ -13,11 +13,10 @@ export const Navigation: React.FC<NavigationProps> = ({ contractible = false, cl
   const { palette, zIndex, spacing } = useTheme();
   const { storefrontConfig } = useStoreon("storefrontConfig");
   const { ownerInfo } = useConfig();
-  let { navigation } = useConfig();
+  const { navigation } = useConfig();
```

Comment:
The navigation here comes from the core api, it's not the specific storefront settings, thus it had the default navigation `configNav` to override this core settings. Overriding as it was done was needed to fix relative routes such as sign in and sign up that needed to be redirected to the portal.

In order to get the settings from the storefront endpoint replace with:
```
const navigation = { ...configNav, ...storefrontConfig.navigation };
```
This way we use the defaults and have them be override with the navigation configuration that comes from the storefront settings.

Reply:
Thanks for the tip Délio! I have updated the code with your suggestions